Drop support for VCD 9.5

Didainius opened the pull request:
Drop support for VCD 9.5 and remove conditional code for API v31.0~32.0

lvirbalas commented on .changes/v2.10.0/544-notes.md:
```diff
@@ -0,0 +1 @@
+* Dropped support for vCD 9.5 [GH-544]
```
Let's use the new capitalization:

```suggestion
* Dropped support for VCD 9.5 [GH-544]
```

vbauzysvmware reviewed (approved):
Seems ok. 
Jus two concerns:

```
		// TODO adjust when we have option to switch between API versions or upgrade the default version
		// API does not return GuestVlanAllowed if API client version is 27.0 (default at the moment) therefore we rely
		// on updating statefile only if the field was returned. In API v31.0 - the field is returned.
		if config.GuestVlanAllowed != nil {
```

also 
```
get_token.sh
runtest.sh
index.html
```
has `version=31.0`

Didainius commented:
> Seems ok.
> Jus two concerns:
> 
> ```
> 		// TODO adjust when we have option to switch between API versions or upgrade the default version
> 		// API does not return GuestVlanAllowed if API client version is 27.0 (default at the moment) therefore we rely
> 		// on updating statefile only if the field was returned. In API v31.0 - the field is returned.
> 		if config.GuestVlanAllowed != nil {
> ```
> 

Fair point. I have removed conditional for this case and doing some testing now. 
> also
> 
> ```
> get_token.sh
> runtest.sh
> index.html
> ```
> 
> has `version=31.0`

Changed

vbauzysvmware commented:
> > Seems ok.
> > Jus two concerns:
> > ```
> > 		// TODO adjust when we have option to switch between API versions or upgrade the default version
> > 		// API does not return GuestVlanAllowed if API client version is 27.0 (default at the moment) therefore we rely
> > 		// on updating statefile only if the field was returned. In API v31.0 - the field is returned.
> > 		if config.GuestVlanAllowed != nil {
> > ```
> 
> Fair point. I have removed conditional for this case and doing some testing now.
> 
> > also
> > ```
> > get_token.sh
> > runtest.sh
> > index.html
> > ```
> > 
> > 
> > has `version=31.0`
> 
> Changed

Don't see commit regarding GuestVlanAllowed

Didainius commented:
> Don't see commit regarding GuestVlanAllowed

Still waiting for `make testnetwork` and `make testvapp` to finish before pushing.

Didainius commented:
> > Don't see commit regarding GuestVlanAllowed
> 
> Still waiting for `make testnetwork` and `make testvapp` to finish before pushing.

They both passed on 9.7 and 10
